app/models/foreman_tasks/concerns/polling_action_extensions.rb

```diff
@@ -0,0 +1,12 @@
+module ForemanTasks
+  module Concerns
+    module PollingActionExtensions
+      def poll_intervals
+        multiplier = Setting[:foreman_tasks_polling_multiplier]
```

Comment:
wouln't this be good just to be on the safe side as we don't have type check for the setting if I read it correctly?
```suggestion
        multiplier = Setting[:foreman_tasks_polling_multiplier].to_f
        multiplier = 1.0 unless multiplier.positive?
```

Reply:
It automatically determines that the setting is an integer and typechecks it as that. We'll always get a non-negative integer here.

Reply:
in that case, ignore me :+1: